e2e/tests/localAuthoritySubmitsApplication_test.js

```diff
@@ -482,3 +482,9 @@ Scenario('local authority submits after giving consent @create-case-with-mandato
   caseViewPage.selectTab(caseViewPage.tabs.documents);
   I.see('council_v_Smith.pdf');
 });
+
+Scenario('local authority confirms payment after submission', async  (I, caseViewPage) => {
+  caseViewPage.selectTab(caseViewPage.tabs.paymentHistory);
+  I.refreshPage(); // Required for some versions of the local build
```

Comment:
Are we leaving it like this?

Reply:
Not sure, doesn't work for me without it and when people update their images surely they will run into the same issue. Strange thing was though I wasn't getting it on preview 🤔